.gitignore

```diff
@@ -13,8 +13,9 @@ addresses.json
 
 # Ignore Rust contract wrappers.
 amm/drink-tests/src/*_contract.rs
+amm/drink-tests/src/psp22.rs
+amm/drink-tests/src/wrapped_azero.rs
```

Comment:
If you're adding these to `.gitignore`, please delete corresponding files from the PR

Reply:
Ah, I will keep them committed into the repo. I'll remove from gitignore. I had problems with `CodeHashMismatch` but it was due to the fact that I had incompatible `*.contract` and `*.json` files.

Reply:
If this is the case then both these files contain strings with paths from your local machine. Please fix this